js/base/Exchange.js

```diff
@@ -947,8 +947,20 @@ module.exports = class Exchange {
         throw new NotSupported (this.id + ' createOrder() is not supported yet');
     }
 
-    cancelOrder (id, symbol = undefined, params = {}) {
-        throw new NotSupported (this.id + ' cancelOrder() is not supported yet');
+    async cancelOrder (id, symbol = undefined, params = {}) {
```

Comment:
Since the base files are not transpiled yet, we will have to add this to py/php as well. Also, I would remove the emulation via cancelOrders, cause 99.9% of the time the exchange will have cancelOrder and will or will not have cancelOrders. I would just make both methods basic stubs:

```JavaScript
    async cancelOrder (id, symbol = undefined, params = {}) {
        throw new NotSupported (this.id + ' cancelOrder() is not supported yet');
    }
```

Reply:
yes, i'm adding them in py/php. but, i have case now (coinFLEX) which have cancelOrder & cancelOrders from same endpoint, where you just have to provide array of ids (doesnt matter if one order id or multiple), so such cases are valid cases of `emulated cancelOrder`. wdyt?

Reply:
Such cases are valid cases for implementing both methods directly without emulation.

Reply:
emulation would have saved manual implementation, but ok, if so, let's close this PR.